roles/dtc/common/tasks/main.yml

```diff
@@ -23,4 +23,4 @@
 
 - name: Import Role Tasks
   ansible.builtin.import_tasks: sub_main.yml
-  tags: "{{ nac_tags.all }}" # Tags defined in roles/common_global/vars/main.yml
+  # tags: "{{ nac_tags.all }}" # Tags defined in roles/common_global/vars/main.yml
```

Comment:
I commented this out because it causes common to run when we try and run the validate role by itself.  Common should not run in this case and since common is a "hidden" infra role we don't call it in isolation so we don't need to tag it